include/mbedtls/mbedtls_config.h

```diff
@@ -1650,6 +1650,18 @@
 */
 #define MBEDTLS_SSL_EARLY_DATA
 
+/**
+ * \def MBEDTLS_SSL_MAX_EARLY_DATA_SIZE
+ *
+ * The maximium amount of 0-RTT data. It only work when MBEDTLS_SSL_EARLY_DATA
+ * is enabled.
+ *
```

Comment:
Requires: needed?

Reply:
No requires.

Reply:
MBEDTLS_SSL_EARLY_DATA depends on this MARCRO

Reply:
is it end?